config/routes.rb

```diff
@@ -47,6 +47,12 @@
             when :get
               root :action => :index
               get "/:c_id", :action => :show
+            when :put
+              put "/:c_id", :action => :update
+            when :patch
+              patch "/:c_id", :action => :update
+            when :delete
+              delete "/:c_id", :action => :destroy
             else
```

Comment:
I think this could be `when :post` instead of `else` just for readability reasons.

Anyway 👍 looks good.